src/components/navBar/navBar.module.scss

```diff
@@ -79,4 +78,106 @@ $theme-active: #87d870;
         color: $white;
         font-weight: 600;
     }
+
+    .hamburgerIcon {
+        display: none;
+    }
+    ul li {
```

Comment:
add this behind a class please.

Reply:
Need to use element selectors, because li and a tag are repetitive in the code and If I use className for li and a tag then it will make the HTML file look boring and dull, I think using element selectors can be useful sometimes in these conditions.

Reply:
this is navbar css, so the CSS applied here will be available throughout the page, leading to problems.
please do the changes.

Reply:
Vinayak, we actually need to discuss this, please let me know your availability so that we can connect to make you understand the scenario

Reply:
Vinayak if you see the HTML file in that li, a, is repeated and if I give them the class name then that would make code look worse. that's why I chose to use element selectors instead of class selectors.
and for further clarifications, let's connect over VC